geth/account/accounts_type.go

```diff
@@ -0,0 +1,62 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
This file should be named `accounts_mock.go`

Reply:
done